pkgs/applications/misc/megasync/default.nix

```diff
@@ -72,7 +70,6 @@ mkDerivation rec {
     ./noinstall-distro-version.patch
     # megasync target is not part of the install rule thanks to a commented block
     ./install-megasync.patch
-    ./ffmpeg_44.patch
```

Comment:
shouldn't we delete the patch file as well if we are not using it?

Reply:
Thanks for reminding me. Removed.